axelrod/tournament.py

```diff
@@ -93,11 +102,15 @@ def play(self, build_results=True, filename=None,
             self.progress_bar = tqdm.tqdm(total=len(self.match_generator),
                                           desc="Playing matches")
 
-        self.setup_output_file(filename)
+        if on_windows and (filename is None):
```

Comment:
Yup: it it takes up a lot of memory then it will crash. It will work fine if a filename is provided though.